great-idea/services.html

```diff
@@ -138,18 +140,20 @@ <h2>Some Facts About Our Services</h2>
             finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris
             euismod facilisis iaculis.
           </p>
-          <ul>
-            <li>Awesome thing</li>
-            <li>Amazing thing</li>
-            <li>Cool thing</li>
-            <li>Great thing</li>
-          </ul>
+          <div class="the-list">
+            <ul>
+              <li>Awesome thing</li>
+              <li>Amazing thing</li>
+              <li>Cool thing</li>
+              <li>Great thing</li>
+            </ul>
           </div>
-        
+        </div>
+
         <img
```

Comment:
What's the best way to style images around text so they hold their resolution?

Reply:
You could give a that image a parent a with a set width of 400px for example. Then give the image a height and width property of 100% and use the object-fit property with a value of cover. This will allow you to make your image parent responsive while making your image responsive/keeping its resolution 

https://medium.freecodecamp.org/time-saving-css-techniques-to-create-responsive-images-ebb1e84f90d5